src/lib/widget/modal.svelte

```diff
@@ -85,6 +86,8 @@
   let _dialogEl: HTMLDialogElement;
   let _journeyComp: SvelteComponent;
 
+  initializeContent(content, true);
```

Comment:
For the Widget (both inline and modal), we essentially take the content provided by the parent app and initialize the content store with it.

Reply:
I like the way this keeps the widget "pure" and leaving those concerns to the parent app. How do we plan to communicate the expectations / assumptions on this content, to developers who implement the widget in their app?

Reply:
e.g. are we going to provide sample translation files with the widget with a complete list of the strings that are required for the callbacks to work, and their counterpart strings built into the AM trees?